bedevere/prtype.py

```diff
@@ -43,7 +43,7 @@ async def classify_by_filepaths(gh, pull_request, filenames):
         if util.is_news_dir(filename):
             news = True
         filepath = pathlib.PurePath(filename)
-        if filepath.suffix == ".rst" or filepath.name == ".nitignore":
+        if filepath.suffix in [".md", ".rst"] or filepath.name == ".nitignore":
```

Comment:
Tiny optimisation (the optimiser might catch this, but stylisticly I'd prefer avoiding membership tests with lists)
```suggestion
        if filepath.suffix in {".md", ".rst"} or filepath.name == ".nitignore":
```